code/datums/martial.dm

```diff
@@ -319,6 +319,90 @@
 		var/mob/living/carbon/human/H = user
 		var/datum/martial_art/plasma_fist/F = new/datum/martial_art/plasma_fist(null)
 		F.teach(H)
-		H << "<span class='notice'>You learn the PLASMA FIST style</span>"
+		H << "<span class='notice'>You read through the scroll and realize you know the secrets of the Plasma Fist.</span>"
+		if(istype(src, /obj/item/weapon/plasma_fist_scroll/oneuse))
+			H.drop_item()
+			visible_message("<span class='warning'>[src] lights up in fire and quickly burns to ash.</span>")
+			new /obj/effect/decal/cleanable/ash(get_turf(src))
+			qdel(src)
 		used = 1
 		desc += "It looks like it's magic was used up."
+
+/obj/item/weapon/plasma_fist_scroll/oneuse
```

Comment:
either you've implemented this completely crappily, or this type name is incredibly wrong.